### PR TITLE
SDST-37 Test Passed

### PR DESCRIPTION
**Pull request by Klik461**

### CODE REVIEW

File: src/test/java/tests/CheckingFilterInTheShopping.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        for (WebElement element : shoppingPage.allPriceOnShoppingPage()) {
+            String text = element.getText().replaceAll("[\\$,]", "");
+            int number = Integer.parseInt(text);
+            numbers.add(number);
+        }
+
+        int min = 50000;
+        int max = 79999;
+
```
Comment (SkipperV) [inline src/test/java/tests/CheckingFilterInTheShopping.java:56]: Remove empty space between variables declaration

**Review by Bogdan27nazaruk — APPROVED** [review]

File: src/test/java/base/AbstractBasePage.java [hunk 2]
```diff
@@ -19,6 +19,7 @@ abstract public class AbstractBasePage {
     protected WebDriver driver;
     protected WebDriverWait wait;
     private int BASE_WAIT = 15000;
+
```
Comment (SkipperV) [inline src/test/java/base/AbstractBasePage.java:22]: Do not need this empty row

File: src/test/java/tests/CheckingFilterInTheShopping.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                .clickBoxMinOrMaxPrice("2")
+                .clickPriceInMinAndMax("79,999")
+                .clickFieldInFilter("Price")
+                .clickFieldInFilter("Model Year")
+                .clickBoxMinOrMaxPrice("1")
+                .clickPriceInMinAndMax("2023")
+                .clickBoxMinOrMaxPrice("2")
+                .clickPriceInMinAndMax("2024");
+
+        List<Integer> numbers = new ArrayList<>();
```
Comment (SkipperV) [inline src/test/java/tests/CheckingFilterInTheShopping.java:46]: Put all declarations to top of test method body

File: src/test/java/tests/CheckingFilterInTheShopping.java (hunk 1 above)
Comment (SkipperV) [reply src/test/java/tests/CheckingFilterInTheShopping.java:56]: Put all declarations to top of test method body

**Review by SkipperV — APPROVED** [review]

**Pull request merged**